docs/units-and-global-variables.rst

```diff
@@ -22,8 +22,8 @@ unit and units are considered naively in the following way:
  * ``1 minutes == 60 seconds``
  * ``1 hours == 60 minutes``
  * ``1 days == 24 hours``
- * ``1 weeks = 7 days``
- * ``1 years = 365 days``
+ * ``1 weeks == 7 days``
```

Comment:
It is one way to turn a spoken language with all its complicated rules into a formal language with less rules ;-)